src/app/frontend/zerostate/zerostate.controller.js

```diff
@@ -30,5 +30,14 @@ export default class ZeroStateController {
     $timeout(() => {
       this.testValue = 8;
     }, 4000);
+
+    // TODO(floreks) think about some place for this
+    /** @export {Array<{title:string, link:string}>} */
```

Comment:
The type of this property should be `{!Array<{...}>}`. The `!` is to indicate that this is non-null reference to an array. 

See jsdoc reference: https://developers.google.com/closure/compiler/docs/js-for-compiler#types

In general, primitives are non-nullable by default and classes are nullable by default. This is a bit non-intuitive, but you'll get used to it :)